src/components/Books/SearchResults.js

```diff
@@ -8,7 +8,7 @@ const SearchResults = props => (
     {props.books.map((book, ix) => (
       <div key={ix} className="book-item-container">
         <BookItem
-          bookId={book.id}
+          bookId={book.id || book.objectID}
```

Comment:
We dont have such a field in our db

Reply:
Don't use this. Maybe you wrote this in firebase Dashboard manually?
Although u can not see the id in firebase dashboard. But it is provided in the fetched data.